src/raftstore/errors.rs

```diff
@@ -150,6 +150,9 @@ impl Into<errorpb::Error> for Error {
                 }
                 errorpb.mut_not_leader().set_region_id(region_id);
             }
+            Error::StoreNotMatch(_to_store_id, _my_store_id) => {
+                errorpb.mut_store_not_match();
```

Comment:
thx,may be some day we need these information to return but not now. Since we would like to make it more easy to process this error in tidb client--just let it clear cache and retry.  So here we needn't return details about expect store id & real store id.